src/plugins/d3/__stories__/line/Shapes.stories.tsx

```diff
@@ -0,0 +1,187 @@
+                    type: 'line',
+                    data: byGenre('2D'),
+                    dataLabels: {
+                        enabled: true,
+                    },
+                },
+                {
+                    name: 'Strategy',
+                    type: 'line',
+                    data: byGenre('Strategy'),
+                    dataLabels: {
+                        enabled: true,
+                    },
+                },
+                {
+                    name: 'Shooter',
+                    type: 'line',
+                    data: byGenre('Shooter'),
+                    dataLabels: {
+                        enabled: true,
+                    },
+                },
+            ],
+        },
+        xAxis: {
+            type: 'datetime',
+            title: {
+                text: 'Release date',
+            },
+        },
+        yAxis: [
+            {
+                title: {text: 'User score'},
+                labels: {
+                    enabled: true,
+                },
+                ticks: {
+                    pixelInterval: 120,
+                },
+            },
+        ],
+    };
+}
+
+const ChartStory = ({data}: {data: ChartKitWidgetData}) => {
+    const [shown, setShown] = React.useState(false);
+
+    if (!shown) {
+        settings.set({plugins: [D3Plugin, HighchartsPlugin]});
+        return <Button onClick={() => setShown(true)}>Show chart</Button>;
+    }
+
+    data.series.data.forEach((graph, i) => {
+        // @ts-ignore
+        graph.dashStyle = SHAPES_IN_ORDER[i % SHAPES_IN_ORDER.length];
+    });
+
+    return (
+        <>
+            <div
+                style={{
+                    height: '80vh',
+                    width: '100%',
+                }}
+            >
+                <ChartKit type="d3" data={{...data, title: {text: 'D3'}}} />
+            </div>
+            <div
+                style={{
+                    height: '80vh',
+                    width: '100%',
+                    marginTop: 20,
+                }}
+            >
+                <ChartKit
+                    type="highcharts"
+                    data={{
+                        data: {
+                            graphs: data.series.data,
+                        },
+                        config: {},
+                        libraryConfig: {
+                            title: {text: 'Highcharts'},
+                            xAxis: {
+                                type: 'datetime',
+                            },
+                        },
+                    }}
+                />
+            </div>
+        </>
+    );
+};
+
+export const ShapesLineChartStory: StoryObj<typeof ChartStory> = {
+    name: 'Shapes',
+    args: {
+        data: prepareData(),
+    },
+    argTypes: {
+        data: {
+            control: 'object',
+        },
+    },
+};
+
+export default {
+    title: 'Plugins/D3/Line',
+    component: ChartStory,
+};
```

Comment:
I think it makes sense to leave such examples clean and make comparisons with other libraries in the Playground story.
I would also like to add this example to the Showcase story